tests/test_alerter.py

```diff
@@ -0,0 +1,12 @@
+from unittest import TestCase
+
+from src import alerter
+
+
+class TestAlerter(TestCase):
+
+    def test_alerter(self):
+        alerter.alert_in_celcius(300.6)
+        alerter.alert_in_celcius(445.8)
+        alerter.alert_in_celcius(45)
+        self.assertEqual(alerter.alert_failure_count, 1)
```

Comment:
This test will pass when any of the above lines increment the `alert_failure_count`. Strengthen the test to assert exactly what you want.